src/_gettsim/transfers/rente.py

```diff
@@ -1817,8 +1817,10 @@ def anteil_entgeltp_ost(
     Proportion of Entgeltpunkte accumulated in East Germany
 
     """
-
-    out = entgeltp_ost / (entgeltp_west + entgeltp_ost)
+    if entgeltp_west + entgeltp_ost == 0:
```

Comment:
```suggestion
    if entgeltp_west == entgeltp_ost == 0.0:
```
Makes clearer what is meant.